### ISSUE TITLE
Theme breaks completely when using "color-scheme prefer-dark"

### ISSUE THREAD

**Issue by Bananaman** [vinceliuice/Fluent-gtk-theme#119]
Hello, I am on the latest GIT code.

The theme becomes completely non-styled if "prefer-dark" is activated.

I am using Fluent-round-orange-Dark. But it seems to happen with all Layan themes.

```
# Break theme completely.
gsettings set org.gnome.desktop.interface color-scheme prefer-dark

# Fix theme.
gsettings set org.gnome.desktop.interface color-scheme default  
```

This is a very important bug, because the "color-scheme" property is used by a TON of modern applications to determine some things:

- What color scheme to use for things like internal syntax highlighting (backgrounds, text colors). To avoid getting black text on dark theme background.
- It is used to hint what colors to use in totally non-GTK applications from other environments such as Qt.

**Comment by ritamsaha00** [comment]
Yes, this becomes a huge problem with Gnome 43 apps in particular
